quipucords/api/common/util.py

```diff
@@ -248,3 +253,20 @@ def expand_scanjob_with_times(scanjob, connect_only=False):
                 status_details[task_key] = task.status_message
 
     return job_json
+
+
+def create_tar_buffer(data_array):
```

Comment:
Is there already a tests_util.py?  Let's make sure and write unit tests for this.

Reply:
I added test for this.